.mailmap

```diff
@@ -1599,6 +1599,7 @@ mao8 <thisisma08@gmail.com>
 mohajain <mohajain99@gmail.com> mohajain <45903778+mohajain@users.noreply.github.com>
 mohammedouahman <simofun85@gmail.com>
 mohit <39158356+mohitacecode@users.noreply.github.com> mohit <42018918+mohitshah3111999@users.noreply.github.com>
+mvg2002 <168417631+mvg2002@users.noreply.github.com>
```

Comment:
shouldn't you add your own mail address here? @mvg2002

Reply:
It should be like:
```
Real Name <myemail@example.com> mvg2002 <168417631+mvg2002@users.noreply.github.com>
```
Then the AUTHORS file will end up showing `Real Name <myemail@example.com>`.

Reply:
My real name and email were already in the mailmap from an old pull request. I had to add this git email as well to pass the check in this PR. I changed it now to:
`Real Name <myemail@example.com> mvg2002 <168417631+mvg2002@users.noreply.github.com>`

Reply:
@mvg2002 it's better to use your personal email. You'll loose access to your student mail at some point.

Reply:
@Tom-van-Woudenberg I had not thought about that, I changed it.